src/app/domains/Company/components/combined/CompanyCombined.template.js

```diff
@@ -27,10 +33,41 @@ const CompanyCombined = (props) => {
         <Box textAlign="center" mt={4}>
           <Button
             type="primary"
-            onClick={() => history.push(ROUTES_PATHS.PROJECT_CREATE)}>
+            onClick={() =>
+              history.push(ROUTES_PATHS.PROJECT_CREATE, {
+                id: companyId,
+                name: data.name
+              })
+            }>
             Add project
           </Button>
         </Box>
+        <Row h="center">
```

Comment:
Why didn't you create separate, reusable component for this part of functionality ?
As i see, there is minimum 3 same code parts, which are repeated...

![image](https://user-images.githubusercontent.com/39984413/113069009-1e228e80-91c8-11eb-8730-53bb19964813.png)